Hackney.Shared.HousingSearch/Domain/Tenure/Tenure.cs

```diff
@@ -8,10 +8,10 @@ namespace Hackney.Shared.HousingSearch.Domain.Tenure
     public class Tenure
     {
         public static Tenure Create(string id, string paymentReference, string startOfTenureDate, string endOfTenureDate,
```

Comment:
Would be good to test that this returns a correct object, including the new props

Reply:
Done in [fe855f2](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/fe855f20214a3db28781b4299853f71c0c033983).